quiz.sql

```diff
@@ -14,22 +17,53 @@ CREATE DATABASE westeros;
 
 -- your code starts here
 
+insert into houses(name, motto, sigil) values('House Baratheon', 'Ours is the fury.', 'http://i.imgur.com/iJtBKIR.png');
+insert into houses(name, motto, sigil) values('House Bolton', 'Our blades are sharp.', 'http://i.imgur.com/uPoc9Jh.jpg');
+insert into houses(name, motto, sigil) values('House Lannister', 'Hear me roar!', 'http://i.imgur.com/gOwmzoU.jpg');
+insert into houses(name, motto, sigil) values('House Martell', '"Unbowed, Unbent, Unbroken."', 'http://i.imgur.com/XaIiVNI.jpg');
+insert into houses(name, motto, sigil) values('House Stark', 'Winter is coming.', 'http://i.imgur.com/zpVjXsa.jpg');
+insert into houses(name, motto, sigil) values('House Targaryen', 'Blood and fire.', 'http://i.imgur.com/4FuSpZt.jpg');
+
+insert into people(name, title, quote, image, age) values('Daenerys Targaryen', 'Queen of Meereen and Mother of Dragons', 'I will take what is mine with blood and fire.', 'http://i.imgur.com/DjHJq4E.jpg', 18);
+insert into people(name, title, quote, image, age) values('Jon Snow', 'Lord Commander of the Nights Watch', '"Night gathers, and now my watch begins."', 'http://i.imgur.com/WadV6VI.jpg', 19);
+insert into people(name, title, quote, image, age) values('Tyrion Lannister', 'Hand of the King and Master of Coin', 'A Lannister always pays his debts.', 'http://i.imgur.com/rNHNWca.jpg', 35);
+insert into people(name, title, quote, image, age) values('Tywin Lannister', 'Wormchow', '"You cannot eat love, nor buy a horse with it, nor warm your halls on a cold night."', 'http://i.imgur.com/ovtHd1H.jpg', 60);
+insert into people(name, title, quote, image, age) values('Arya Stark', 'Princess of Winterfell', '...anyone can be killed.', 'http://i.imgur.com/Csq9exg.jpg', 14);
+insert into people(name, title, quote, image, age) values('Cersei Lannister', 'King Mother and Lady of Casterly Rock', '"The gods have no mercy, that’s why they’re gods."', 'http://i.imgur.com/jYeeCJ3.jpg', 40);
+insert into people(name, title, quote, image, age) values('Robb Stark', 'Deceased', 'It was the only honorable thing to do.', 'http://i.imgur.com/uYC3zzt.jpg', 21);
+
+
+
 -- your code ends here
 
 -- Question 3. Write a query to get all attributes of people greater than a certain age *or* having a specific title.
 
 -- your code starts here
 
+SELECT id, name, title, quote, image, age
```

Comment:
You can use '*' in place of a list of all of the attributes here. Also, you can combine both of these queries into one like so:

`SELECT * FROM people WHERE age > 2 OR name = 'Princess of Winterfell';`